cclib/parser/orcaparser.py

```diff
@@ -144,54 +144,77 @@ def extract(self, inputfile, line):
                 elif line[0] == '%':
                     pass
 
-                # Geometry block
```

Comment:
Where did this block run off to?

Reply:
Sorry I should have put a comment about this. This block parses the geometry _only_ for storing in the metadata. This isn't done in any of the other parsers, and I didn't see the use of it so I removed it. If this is something that is desired, then I could add it back in. It would have to be generalized a bit since it is incompatible with parameter scan calculations.